src/DateUtils.js

```diff
@@ -16,39 +16,51 @@ limitations under the License.
 
 'use strict';
 
-var days = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"];
-var months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
+import UserSettingsStore from './UserSettingsStore';
+const days = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"];
+const months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
+let time;
 
 function pad(n) {
     return (n < 10 ? '0' : '') + n;
 }
 
+function twentyfourhour(date) {
```

Comment:
please indent consistently as per the code-style (i.e. 4 spaces).  The function probably should be `twentyFourHour()` too for consistent casing.